### PR TITLE
Csv output

### PR DESCRIPTION
**Pull request by SadCheeto**
## Describe your changes
Added creation of csv files. 
VERY basic and needs to be changed later on
## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] Have I requested and notified the boys for review of the pull-request?

### CODE REVIEW

File: byoqm/models/code_climate.py [hunk 1]
```diff
@@ -21,6 +25,18 @@ def getDesc(self) -> Dict:
...
+        file_location = path + "/" + str(date.today()) + ".csv"
+        if not os.path.exists(path):
+            os.mkdir(path)
+        with open(file_location, "w") as file:
+            writer = csv.writer(file)
+            writer.writerow(["Metric", "Value"])
+            writer.writerow(["Argument Count", str(self.argument_count())])
+            writer.writerow(["File Length", str(self.file_length())])
+            writer.writerow(["Method Count", str(self.method_count())])
+            writer.writerow(["Return Statements", str(self.return_statements())])
```
Comment (toffernator) [inline byoqm/models/code_climate.py:38]: I think it makes most sense to iterate over the `.getDesc()` so we don't have to update this by hand

File: output/2023-02-22.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Metric,Value
```
Comment (toffernator) [inline output/2023-02-22.csv:1]: I think we should consider .gitignore the outputs of the program.

File: byoqm/models/code_climate.py (hunk 1 above)
Comment (SadCheeto) [reply byoqm/models/code_climate.py:38]: i agree - This was just for getting some output that freddy could use for his visualization

File: output/2023-02-22.csv (hunk 2 above)
Comment (Kobo-coder) [reply output/2023-02-22.csv:1]: +1

**Review by toffernator — APPROVED** [review]

**Pull request merged**